### ISSUE TITLE
ignition v0.24.0: failed to acquire config: context canceled, during pxe boot

### ISSUE THREAD

**Issue by goswyperdev** [coreos/bugs#2436]
I am using the following ignition config file with pxe boot:

IGNITION CONFIG
```json
{
  "ignition": {
    "config": {},
    "timeouts": {},
    "version": "2.1.0"
  },
  "networkd": {},
  "passwd": {
    "users": [
      {
        "name": "core",
        "passwordHash": "$1$INuWtFQI$DkbOpOthzxrDvrPRPNgU5/",
        "sshAuthorizedKeys": [
          "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDGdByTgSVHq..."
        ]
      }
    ]
  },
  "storage": {
     "files": [
      ],
      "directories": [
      ]
  },
  "systemd": {
    "units": [
    ]
  }
}
```

PXE CONFIG
```
default coreos
prompt 1
timeout 15

display boot.msg 

label coreos
  menu default
  kernel netboot/coreos_production_pxe.vmlinuz
  initrd netboot/coreos_production_pxe_image.cpio.gz
  append coreos.first_boot=1 ip=dhcp coreos.autologin=tty1 coreos.config.url=http://192.168.8.222/infiboot/configs/pxe-config.ign
```

During pxe boot I see that http request succeeds but still seeing these errors:
```
ignition[605]: no config at "/usr/lib/ignition/base.ign"
ignition[605]: GET http://192.168.8.222/infiboot/configs/pxe-config.ign attempt #1 
ignition[605]: GET result : OK
ignition[605]: failed to fetch config: context canceled
ignition[605]: failed to acquire config: context canceled
```

I check ignition code and could not understand how fetch/acquire config fails after GET result is OK. I understand 'context canceled' is coming from GO lang lib. I am new to GO lang and could not grasp the context functionality properly.

Please help me understand in what cases fetch could fail after GET succeeds.

**Comment by lucab** [comment]
This is a duplicate of https://github.com/coreos/bugs/issues/2435, which has been fixed in alpha 1786.0.1 and beta 1772.1.1.

Which ContainerLinux release are you using?

**Comment by goswyperdev** [comment]
Hi Folks,
Thanks for a prompt response, picking the latest Alpha channel fixed the issue. I did lookup before opening the bug, could not find the issue for some reason.

Thanks once again.
